include/aws/crt/mqtt/MqttClient.h

```diff
@@ -460,6 +468,23 @@ namespace Aws
                     const Io::SocketOptions &socketOptions,
                     bool useWebsocket) noexcept;
 
+                MqttConnection(
+                    aws_mqtt5_client *client,
```

Comment:
Should we rename this to mqtt5Client so that we don't accidentally confuse it with the `aws_mqtt_client` that's being used in the other constructors?